bundles.json

```diff
@@ -23,7 +23,7 @@
     "platform/features/events",
     "platform/forms",
     "platform/identity",
-    "platform/persistence/local",
+    "platform/persistence/elastic",
```

Comment:
Don't believe we want to include this change - think local storage should still be default persistence store for open source